### ISSUE TITLE
Format JSON body doesn't work when property names contain colons

### ISSUE THREAD

**Issue by ffxsam** [rangav/thunder-client-support#345]
**Describe the bug**
When JSON property names contain colons, the format feature doesn't work.

**To Reproduce**
Paste this into the Json Content:

```json
{
"test::ok":true,
"also:bad":"hi"
}
```

Then click "Format".

**Expected behavior**
I'd expect it to auto-indent and fix spacing.

**Platform:** 
 - OS: macOS 11.6
 - vscode version: 1.60.2
 - extension version: 1.9.1

**Comment by rangav** [comment]
Thanks @ffxsam for reporting the bug, will verify and fix it.

**Comment by ffxsam** [comment]
Actually, I noticed this bug also happens when there is a colon in the value as well.

```json
{
"test": "this:is:bad"
}
```

**Comment by rangav** [comment]
This is now fixed in v1.10.0
https://github.com/rangav/thunder-client-support/releases/tag/v1.10.0

please let me know if any feedback
